react-exercise/src/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import Overview from "../../components/user-profile-overview/Overview.tsx";
+import Posts from "../../components/user-profile-posts/Posts.tsx";
+import { User } from "../../types/user.ts";
+import "./Tabs.scss";
+import clsx from "clsx";
+
+interface TabsProps {
+  user: User;
+}
+
+type TabOption = "overview" | "posts";
```

Comment:
WDYT about using a type here? I was a bit confused in general about how to implement the tabs. Is the fact that it's not generic and the info is decided inside the component is reasonable? Is it ok that the content that each tab is presenting is also decided in here in this manner? If there is a better approach I will like to hear

Reply:
Also, if there is a regular tabs library for these kind of implementation needs, generally used in the LTX studio project, please let me know :)

Reply:
Lets discuss F2F